test TestVerifyOdsProjectProvisionThruProvisionApi has been failing since 2020-08-25 12:00

**Describe the bug**
Running test TestVerifyOdsProjectProvisionThruProvisionApi on ODS box will fail with the error message as listed below.

**To Reproduce**
Steps to reproduce the behavior:
1. Create new ODS box
2. run test TestVerifyOdsProjectProvisionThruProvisionApi
3. See error

**Expected behavior**
test will pass

**Affected version (please complete the following information):**
 - OpenShift: 3.11
 - OpenDevStack latest master

**Log Output (ensure to remove any confidential information like tokens, project names, etc.**
```
    amazon-ebs: === RUN   TestVerifyOdsProjectProvisionThruProvisionApi
    amazon-ebs: --- FAIL: TestVerifyOdsProjectProvisionThruProvisionApi (5.06s)
    amazon-ebs:     verify-ods-project-provision-api_test.go:40: Failed to delete project: Execution of `provisioning-app-api.sh` for 'DELETE/ODSVERIFY' failed:
    amazon-ebs:         StdOut: No config file found, assuming defaults, current dir: /home/openshift/opendevstack/ods-core/tests/ods-verify
    amazon-ebs:
    amazon-ebs:         Started provision project/component script with command (DELETE)!
    amazon-ebs:
    amazon-ebs:         ... encoding basic auth credentials in base64 format
    amazon-ebs:
    amazon-ebs:         ... sending request to 'https://prov-app-ods.ocp.odsbox.lan' (output will be saved in file './response.txt' and headers in file './headers.txt')
    amazon-ebs:
    amazon-ebs:         delete / get project - DELETE
    amazon-ebs:         curl DELETE request successful...
    amazon-ebs:
    amazon-ebs:         ... displaying HTTP response body (content from './response.txt'):
    amazon-ebs:         No request (body) response recorded
    amazon-ebs:
    amazon-ebs:         ... displaying HTTP response code
    amazon-ebs:         http_resp_code=404
    amazon-ebs:
    amazon-ebs:         something went wrong... endpoint responded with error code [HTTP CODE=404] (expected was 200)
    amazon-ebs:
    amazon-ebs:         StdErr:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
    amazon-ebs:                                          Dload  Upload   Total   Spent    Left  Speed
    amazon-ebs:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0* About to connect() to prov-app-ods.ocp.odsbox.lan port 443 (#0)
    amazon-ebs:         *   Trying 172.31.2.125...
    amazon-ebs:         * Connected to prov-app-ods.ocp.odsbox.lan (172.31.2.125) port 443 (#0)
    amazon-ebs:         * Initializing NSS with certpath: sql:/etc/pki/nssdb
    amazon-ebs:         * skipping SSL peer certificate verification
    amazon-ebs:         * SSL connection using TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
    amazon-ebs:         * Server certificate:
    amazon-ebs:         *     subject: CN=*.ocp.odsbox.lan
    amazon-ebs:         *     start date: Aug 25 10:11:30 2020 GMT
    amazon-ebs:         *     expire date: Aug 25 10:11:31 2022 GMT
    amazon-ebs:         *     common name: *.ocp.odsbox.lan
    amazon-ebs:         *     issuer: CN=openshift-signer@1598350104
    amazon-ebs:         > DELETE /api/v2/project/ODSVERIFY HTTP/1.1
    amazon-ebs:         > User-Agent: curl/7.29.0
    amazon-ebs:         > Host: prov-app-ods.ocp.odsbox.lan
    amazon-ebs:         > Authorization: Basic b3BlbnNoaWZ0Om9wZW5zaGlmdA==
    amazon-ebs:         > Accept: application/json
    amazon-ebs:         > Content-Type: application/json
    amazon-ebs:         >
    amazon-ebs:   0     0    0     0    0     0      0      0 --:--:--  0:00:04 --:--:--     0< HTTP/1.1 404
    amazon-ebs:         < Set-Cookie: JSESSIONID=54EB12E95C65C0A52232CCD788BDFE23; Path=/; Secure; HttpOnly
    amazon-ebs:         < X-Content-Type-Options: nosniff
    amazon-ebs:         < X-XSS-Protection: 1; mode=block
    amazon-ebs:         < Cache-Control: no-cache, no-store, max-age=0, must-revalidate
    amazon-ebs:         < Pragma: no-cache
```

--- a/tests/scripts/provisioning-app-api.sh
+++ b/tests/scripts/provisioning-app-api.sh
@@ -89,9 +89,12 @@ echo
 echo "... displaying HTTP response code"
 echo "http_resp_code=${http_resp_code}"
 echo
-if [ $http_resp_code != 200 ]
-  then
-    echo "something went wrong... endpoint responded with error code [HTTP CODE="$http_resp_code"] (expected was 200)"
-    exit 1
+if [ $http_resp_code != 200 ]; then
+	if [ ${COMMAND} == "DELETE" ] && [ ${http_resp_code} == 404 ]; then
+		echo "... DELETE request responded with 404 - continuing as resource does not exist"
+	else
+		echo "something went wrong... endpoint responded with error code [HTTP CODE="$http_resp_code"] (expected was 200)"
+		exit 1
+	fi
 fi
 echo "provision project/component request (${COMMAND}) completed successfully!"

